game/scripts/vscripts/components/boss/ngp.lua

```diff
@@ -5,8 +5,13 @@ if NGP == nil then
     NGP = class({})
 end
 
-NGP.itemIndex = 0
+NGP.itemIndex = 2
 NGP.activeItems = {}
+NGP.activeTimers = {}
+local totalgoodplayers = 0
+local totalbadplayers = 0
+local ngpItemsGood = {}
+local ngpItemsBad = {}
```

Comment:
I'm not sure this is actually important, but I feel like all these file-scope local variables ought to be properties on the NGP class like `itemIndex` and `activeItems` are.